src/librustdoc/html/static/rustdoc.css

```diff
@@ -1754,16 +1729,56 @@ div.name.expand::before {
 	top: 2px;
 }
 
-/* This part is to fix the "Expand attributes" part in the type declaration. */
-.type-decl > pre > .toggle-wrapper.toggle-attributes.top-attr {
-	margin-left: 0 !important;
+/* The hideme class is used on summary tags that contain a span with
```

Comment:
People use `<details>` in doc contents as well, we absolutely should be using `details.someclass` here